theforeman/foreman
# Fixes #26727 - Fix issue with primary interface on bond

Prerequisites for testing:
* Host with bond interface as primary interface

Tests needed (anticipated impacts):
* Edit host with bond

Steps to Reproduce:
1. create a host with a bond0 interface mode=0 with the ip address and add one slave (enough to test) 
2. delete the host in foreman 
3. run puppet to upload the facts, this should create through the puppet ENC the host in foreman
4. check the interfaces of the host, the bond0 is missing

We have this issue open with several customers since quite a while; Therefore I'm now proposing this directly here. Please let me know if something needs to be adapted.
